src/features/AddAccount/AddAccountForm.reducer.tsx

```diff
@@ -21,7 +21,7 @@ export const formReducer = (formData: FormData, action: FormDataAction) => {
       const accountAndDerivationPath: IAccountAdditionData[] = handleUnlock(
         formData.accountType,
         action.payload
-      );
+      ).map((a: any) => ({ ...a, dPath: a.derivationPath || a.dPath }));
```

Comment:
Since there is a helper, move to `handleUnlock` so its response type is strongly typed.